python/paddle/v2/framework/tests/op_test.py

```diff
@@ -295,15 +295,16 @@ def find_actual(target_name, fetch_list):
                             + ") has different lod at " + str(place))
             else:
                 idx = find_actual(out_name, fetch_list)
-                actual_t = outs[idx]
+                actual = outs[idx]
+                actual_t = np.array(actual)
```

Comment:
what is the purpose of these changes?

Reply:
Because type of outs[idx] is LoDTensor, not numpy array. We should convert it to numpy array before comparing.